interface/python_binding/pyktx/gl_internalformat.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2021, Shukant Pal and Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+from enum import IntEnum
+
+
+class GlInternalformat(IntEnum):
```

Comment:
Is this a complete list of all the OpenGL internal formats?

Reply:
There is a 1:1 match between these formats and the ones defined in the Java interface.